client_packages/chat/js/chat.js

```diff
@@ -0,0 +1,172 @@
+const BUTTON_T = 84,
+    BUTTON_ENTER = 13,
+    BUTTON_ESCAPE = 27,
+    BUTTON_ARROW_UP = 38,
+    BUTTON_ARROW_DOWN = 40;
+
+let chat = {
+    size: 0,
+    container: null,
+    input: null,
+    enabled: false,
+    active: true,
+    timer: null,
+    previous: new Array(),
+    messageNumber: 0,
+    hide_chat: 15*1000 // 15 - seconds
+};
+var langRegex = {
+    "EN": /^[a-zA-Z]+$/,
+    "RU": /[а-яА-ЯёЁ]/,
+    "UA": /[ієїґ\']+/ig
+};
+function enableChatInput(enable)
+{
+    if(chat.active == false
+        && enable == true)
+        return;
+    if (enable != (chat.input != null))
+    {
+        mp.invoke("focus", enable);
+        if (enable)
+        {
```

Comment:
move `{` to the previous line: `if (enable != (chat.input != null)) {`
And please formate all files, ctrl+alt+l on windows